models/demos/roberta/demo/demo.py

```diff
@@ -250,44 +255,45 @@ def run_roberta_question_and_answering_inference_squad_v2(
             i += 1
         eval_score = squad_metric.compute(predictions=pred_labels, references=true_labels)
         cpu_eval_score = squad_metric.compute(predictions=cpu_pred_labels, references=true_labels)
-        logger.info(f"\tTT_Eval: exact: {eval_score['exact']} --  F1: {eval_score['f1']}")
+        logger.info(f"TT_Eval: exact: {eval_score['exact']} --  F1: {eval_score['f1']}")
```

Comment:
We should probably assert on the scores, otherwise this test doesn't tell us the model is working

Reply:
I've updated the demo asserting the scores.

Reply:
`output_start_logits` is index into the array where input context is stored, so PCC is low here as we are comparing single numbers, not tensors. 
@kkeerthana0573 what would make sense is to also assert on `eval_score` as this will be a better measure if model is working.